module/library/individualAssessment/helper.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * name : helper.js
+ * author : Aman
+ * created-date : 04-Jun-2020
+ * Description : Individual library helper functionality.
+ */
+
+// Dependencies 
+const solutionsHelper = require(MODULES_BASE_PATH + "/solutions/helper");
+
+ /**
+    * IndividualAssessmentHelper
+    * @class
+*/
+
+module.exports = class IndividualAssessmentHelper {
+
+    /**
+      * List of individual solution templates.
+      * @method
+      * @name list
+      * @param {String} search - Search Data.
+      * @param {Number} limit - limitting value.
+      * @param {Number} pageNo
+      * @returns {Object} returns list of individual solutions
+     */
+
+    static list(search,limit,pageNo) {
+        return new Promise(async (resolve, reject) => {
+            try {
+                
+                let individualSolution = 
+                await solutionsHelper.templates( 
+                  messageConstants.common.INDIVIDUAL,
+                  search, 
+                  limit, 
+                  pageNo,  
+                );
+
+                return resolve({
+                  message : messageConstants.apiResponses.INDIVIDUAL_SOLUTIONS_LIST,
+                  result : individualSolution
+                });
+            } catch (error) {
+                return reject({
+                    status: error.status || httpStatusCode.internal_server_error.status,
+                    message: error.message || httpStatusCode.internal_server_error.message,
+                    errorObject: error
+                });
+            }
+        })
+    }
+
+    /**
+      * Individual solution information
+      * @method
+      * @name details
+      * @param {String} templateId - Template id.
+      * @returns {Object} returns creator,about and questions.
+     */
+
+    static details(templateId) {
+        return new Promise(async (resolve, reject) => {
+            try {
+
+              let individualSolutionDetails = 
+              await solutionsHelper.solutionAndcriteriaDetails(
+                templateId
+              );
+
+              let getQuestionFromEcm = await solutionsHelper.getQuestionFromEcm(
+                individualSolutionDetails.criteriaDocuments,
```

Comment:
@aman-tunerlabs - Use the question key - "showQuestionInPreview" to list questions in template